model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/solve_nonhydro.py

```diff
@@ -311,7 +312,26 @@ def __init__(
         #: parameters from other namelists:
 
         #: from mo_interpol_nml.f90
-        self.nudge_max_coeff: float = max_nudging_coeff
+
```

Comment:
Can you add a `# TODO` that this should be handled in the configuration utility, once we get to this, such that we can get rid of the duplication.